tests/baselines/reference/tscWatch/programUpdates/Updates-diagnostics-when-'--allowArbitraryExtensions'-changes.js

```diff
@@ -0,0 +1,181 @@
+Input::
+//// [/a/lib/lib.d.ts]
+/// <reference no-default-lib="true"/>
+interface Boolean {}
+interface Function {}
+interface CallableFunction {}
+interface NewableFunction {}
+interface IArguments {}
+interface Number { toExponential: any; }
+interface Object {}
+interface RegExp {}
+interface String { charAt: any; }
+interface Array<T> { length: number; [n: number]: T; }
+
+//// [/a.ts]
+import {} from './b.css'
+
+//// [/b.d.css.ts]
+declare const style: string;
+
+//// [/tsconfig.json]
+{"compilerOptions":{"allowArbitraryExtensions":true}}

```

Comment:
This needs to have "a.ts" as files so that "b.d.css.ts" is not included by default ?

Reply:
`--explainFiles` should show you why the file is in program

Reply:
Added an explicit files list.

Reply:
Hah, this is what I didn’t do in my local test